.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Deploy Docs
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
`Setup Node.js`

Reply:
I don't get it? Yes, we're setting up node.js

Reply:
Don't get it .. yes we're setting up a newer node.js!

Reply:
You se `name: ` for other steps but need to set for this one as well.